util/measurementReportUtils/codeSequences.py

```diff
@@ -11,13 +11,13 @@ class Finding(CodeSequence):
 
   def __init__(self, segmentedStructure):
     if segmentedStructure == "NormalROI_PZ_1":
-      super().__init__("Normal", "SRT", "G-A460")
+      super().__init__("Normal", "SCT", "17621005")
     elif segmentedStructure == "PeripheralZone":
-      super().__init__("Entire", "SRT", "R-404A4")
```

Comment:
Just noticed that some SRT codes haven't been replaced. I've used [pydicom snomed dictionary](https://github.com/pydicom/pydicom/blob/master/pydicom/sr/_snomed_dict.py). looks like it is missing some codes "R-404A4".

Reply:
This is pretty much the list of missing (not replaced) srt sct codes in the project:
"T-48440", "T-47741", "T-40210", "T-1416B", "T-AB000", "T-4806E", "R-102BC", "A-13510", "T-70010", "T-141A5", "T-11500", "T-49423", "T-A0193", "T-F6078", "T-D05E4", "T-4881F", "T-4882A", "T-9200B", "T-45526", "A-26A06", "R-404A4", "T-14183", "T-41040", "T-43126", "A-10044", "T-51000", "T-48470", "D4-31320", "T-48052", "A-26A08", "T-41070", "T-00317", "T-48817", "T-4942C", "T-41066", "T-1A310"

Reply:
Can you check those against the standard here: http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_O.html#table_O-1 ?

Reply:
Checked the page, it doesn't have either. There is one other [page](http://dicom.nema.org/medical/dicom/2016b/output/chtml/part16/sect_CID_3010.html) though, that has some of them.

Reply:
Please work against [current version of the standard](http://dicom.nema.org/medical/dicom/current/output/chtml/part16/sect_CID_3010.html). The one you referenced is from 2016.

Reply:
Well, the reason that Google found the older versions was that they are deprecated values. I could find majority of missing labels [here](http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_J.html).